doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
```

Comment:
Can you give a more general introduction what is mounting for and that it can be used for many files that build up a global hierarchy?